### PR TITLE
Backwards compatibility test for MemNN (#1881)

### PR DESCRIPTION
**Pull request by domrigoglioso**
**Patch description**
Related Issue: #1881
 Trained a MemNN model on `-t integration_test`
(Full command: 
`python examples/train_model.py -m memnn -t integration_tests -mf /tmp/integ_memnn -bs 1 -sval True -vtim 1000 -ttim 7200`)

**Testing steps**
Ran test_memnn.py and expected to pass test cases (with one skip). Test cases passed.

**Logs**
```
$ python tests/test_memnn.py

.s.
----------------------------------------------------------------------
Ran 3 tests in 16.772s

OK (skipped=1)
```

### CODE REVIEW

File: tests/test_memnn.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+# !/usr/bin/env python3
```
Comment (stephenroller) [inline tests/test_memnn.py]: No space here.

File: tests/test_memnn.py [hunk 2]
```diff
@@ -77,6 +77,39 @@ def test_labelcands_multi(self):
             "test hits@1 = {}\nLOG:\n{}".format(test['hits@1'], stdout),
         )
 
+    def test_backcomp(self):
+        '''
```
Comment (stephenroller) [inline tests/test_memnn.py]: use " instead of ' for docstring quotes.

File: tests/test_memnn.py [hunk 3]
```diff
@@ -77,6 +77,39 @@ def test_labelcands_multi(self):
...
+                model='memnn',
+                model_file='zoo:unittest/memnn/model',
+                dict_file='zoo:unittest/memnn/model.dict',
+                batch_size=16,
+            )
+        )
+
+        self.assertGreaterEqual(
+            valid['accuracy'],
+            0.88,
```
Comment (stephenroller) [inline tests/test_memnn.py]: was hoping for a higher number but alright.

Comment (domrigoglioso) [reply tests/test_memnn.py]: I could try re-training the model with a different learning rate/other parameters or for a longer time if necessary. Also Kurt has already uploaded the model.

Comment (stephenroller) [reply tests/test_memnn.py]: dope, this is perfect then.

**Pull request merged**